LibertyDSNP/example-client
# Feature/metamask how to/178789849

Purpose
---------------
feature: add how-to guide to setting up metamask and torus


Solution
---------------
add instructions text and styling


Change summary
---------------
* added text
* met with heather to discuss styling
* implemented styling


Steps to Verify
----------------
1. run code
2. make sure that the directions are descriptive enough.


Additional details / screenshot
----------------
- Any suplmenental pictures or material
- ![Screenshot]()
